2024-08-18 03:57:34.218391-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:29)
2024-08-18 03:57:35.135375-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:34)
2024-08-18 03:57:35.642425-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:35)
2024-08-18 04:01:37.070632-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:29)
2024-08-18 04:01:38.143849-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:34)
2024-08-18 04:01:38.998806-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:35)
2024-08-18 04:04:55.445442-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:29)
2024-08-18 04:04:56.600488-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:34)
2024-08-18 04:04:56.900566-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:35)
2024-08-18 04:06:25.899582-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:29)
2024-08-18 04:06:26.591605-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:34)
2024-08-18 04:06:26.908949-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:35)
2024-08-18 04:08:21.987480-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:30)
2024-08-18 04:08:22.734785-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:35)
2024-08-18 04:08:23.026908-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:36)
2024-08-18 04:09:17.825426-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:31)
2024-08-18 04:09:18.836545-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:36)
2024-08-18 04:09:19.314682-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:37)
2024-08-18 04:10:05.763197-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:31)
2024-08-18 04:10:06.569723-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:36)
2024-08-18 04:10:06.875181-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:37)
2024-08-18 04:10:57.240409-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:31)
2024-08-18 04:10:57.830606-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:36)
2024-08-18 04:10:58.543230-06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "2" not found [42122-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:39)
	at dao.implementation.PacienteImplDaoH2.selectPorId(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.selectPorId(PacienteServicio.java:16)
	at Main.main(Main.java:37)
2024-08-18 04:12:03.007560-06:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]" [50000-230]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:344)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:682)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at dao.BD.getConnection(BD.java:21)
	at dao.BD.crearTabla(BD.java:43)
	at Main.main(Main.java:10)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]" [50000-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 14 more
2024-08-18 04:12:03.219191-06:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]" [50000-230]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:344)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:682)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at dao.BD.getConnection(BD.java:21)
	at dao.implementation.PacienteImplDaoH2.insertRegistro(PacienteImplDaoH2.java:111)
	at dao.implementation.PacienteImplDaoH2.insertRegistro(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.insertRegistro(PacienteServicio.java:20)
	at Main.main(Main.java:14)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]" [50000-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 16 more
2024-08-18 04:12:03.268196-06:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]" [50000-230]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:344)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:682)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at dao.BD.getConnection(BD.java:21)
	at dao.implementation.PacienteImplDaoH2.updateRegistro(PacienteImplDaoH2.java:153)
	at dao.implementation.PacienteImplDaoH2.updateRegistro(PacienteImplDaoH2.java:12)
	at service.PacienteServicio.updateRegistro(PacienteServicio.java:24)
	at Main.main(Main.java:20)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]" [50000-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /home/iseddev/Documents/dh_techdeveloper/5TO_BIM/Backend_I/Clases/C11_Repaso/C11S_Repaso_Clase/paciente_bd.mv.db [2.3.230/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 16 more
